src/cargo/core/summary.rs

```diff
@@ -403,25 +356,35 @@ impl FeatureValue {
         }
     }
 
-    /// Returns `true` if this feature explicitly used `dep:` syntax.
-    pub fn has_dep_prefix(&self) -> bool {
-        matches!(self, FeatureValue::Dep { .. })
+    fn explicitly_name(&self) -> Option<InternedString> {
```

Comment:
Here `name` seems ambiguous to me. It could be a "feature name" or a "dependency name". I might go with something like `explicit_dep_name` or something like that.

Also, can you add a docstring to say exactly what it does?